lib/index.d.ts

```diff
@@ -138,11 +136,21 @@ export function decryptMIMEMessage(
     signatures: OpenPGPSignature[];
 }>;
 
-export interface EncryptOptionsPmcryptoWithData extends Omit<EncryptOptions, 'message'> {
-    data: Uint8Array | string;
-    returnSessionKey?: boolean;
+type MaybeStream<T extends Uint8Array | string> = T | ReadableStream<T>;
+export interface EncryptOptionsPmcryptoWithTextData extends Omit<EncryptOptions, 'message'> {
+    textData: MaybeStream<string>;
+    binaryData?: undefined;
+    stripTrailingSpaces?: boolean;
```

Comment:
Are we leaving this here, or are we requiring the user to clean the input themselves if needed?
Same question for sign/verifyMessage.
Note this is not the same as a `asCleartextMessage` flag, since the created message is always of type `Message`.

Reply:
Typescript doesn't allow setting both options though

Reply:
Right, ok. That's good at least :+1: I still think it would be more obvious to the apps why that is if they had to call the function themselves, but yeah, we can also have it as an option.

Reply:
I'd keep it for now and remove it before release based on how much is needed in the apps. Also so that if we do remove it, we'll have a more visible commit explicitly removing support for trimming in the history.